source/aws_profile.c

```diff
@@ -556,10 +556,8 @@ static int s_profile_merge(struct aws_profile *dest_profile, const struct aws_pr
                 return AWS_OP_ERR;
             }
 
-            struct aws_byte_cursor empty_value;
-            AWS_ZERO_STRUCT(empty_value);
-
             struct aws_byte_cursor property_name = aws_byte_cursor_from_string(dest_key);
+            static struct aws_byte_cursor empty_value = AWS_BYTE_CUR_INIT_FROM_STRING_LITERAL("");
             dest_property = aws_profile_property_new(dest_profile->allocator, &property_name, &empty_value);
```

Comment:
weird.
you said you were working around  a crash in `aws_profile`
we should probably just fix that so it can take cursor with NULL ptr

Reply:
Determined this is just an issue with my environment. Reverting for now, will come back with a better fix later.